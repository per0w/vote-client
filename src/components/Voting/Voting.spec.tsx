import React from 'react';
import { shallow } from 'enzyme';

import Voiting from './index';


describe('Voting', () => {
  it('renders a pair of buttons', () => {
    const buttons = shallow(<Voiting pair={['Bleach', 'Fairy Tail']} />).find('button');
    expect(buttons).toMatchSnapshot();
    expect(buttons.length).toEqual(2);
  });
});
