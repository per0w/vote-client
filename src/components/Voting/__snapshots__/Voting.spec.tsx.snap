// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voting renders a pair of buttons 1`] = `
Array [
  <button
    key="Bleach"
    type="button"
  >
    <h1>
      Bleach
    </h1>
  </button>,
  <button
    key="Fairy Tail"
    type="button"
  >
    <h1>
      Fairy Tail
    </h1>
  </button>,
]
`;
